packages/blocks/ternary/src/index.ts

```diff
@@ -15,10 +15,22 @@ export const ternary = createBlock({
   actions: [
     getDataIntegrations,
     getBudgets,
-    // forecasting
-    // cost allocation?
-    //getUsageRecommendations,
-    //updateUsageRecommendations,
+    createCustomApiCallAction({
+      baseUrl: (auth: any) => `${auth.apiURL}/api/`,
+      auth: ternaryCloudAuth,
+      additionalProps: {
+        documentation: Property.MarkDown({
+          value:
+            'For more information, visit the [Ternary API documentation](https://docs.ternary.app/reference/api).',
+        }),
+      },
+      additionalQueryParamsMapping: async ({ auth }: any) => ({
```

Comment:
according to ternary docs, you have to pass the tenant id as a query param in the requests.